assets/prototype/domain/eventEditor/data/mutations/tickets/updatePriceCache.ts

```diff
@@ -19,24 +19,30 @@ const updatePriceCache = ({ proxy, prices = {}, ticketIn, ticketId, remove = fal
 	try {
 		data = proxy.readQuery(queryOptions);
 	} catch (error) {
-		data = {};
+		data = {
+			espressoPrices: DEFAULT_ENTITY_LIST_DATA('Prices'),
```

Comment:
I was expecting `DEFAULT_ENTITY_LIST_DATA` to be a constant/ object/ enum, not a function :)

Reply:
Initially I defined it as a constant but then thought it's too much to define that for each entity.